docker-compose.yml

```diff
@@ -141,6 +141,7 @@ services:
       - PRODUCT_CATALOG_ADDR
       - SHIPPING_ADDR
       - KAFKA_ADDR
+      - GOMEMLIMIT=16MiB
```

Comment:
Could you share more context about the 16MiB value? Is the container's memory limit around 20MiB?

Reply:
Great! Missed the limits above.